components/form/src/ErrorMessage.js

```diff
@@ -43,17 +35,17 @@ export const ErrorMessage = ({ message, icon, tag: Tag, ...props }) => {
 	}
 
 	const common = {
-		fontSize: errorMessage.fontSize,
-		margin: errorMessage.margin,
-		color: errorMessage.color,
+		fontSize: '0.875rem',
+		margin: '0 0 0.75rem',
+		color: COLORS.danger,
 		...(isMessages && { listStyle: 'none', paddingLeft: 0 }),
```

Comment:
can do in one line
```
...(Array.isArray(message) && { listStyle: 'none', paddingLeft: 0 }), 
```

Reply:
It's used above though, saves calling isArray twice... thought the variable and comment also made it clearer

Reply:
Didnt see it being used above, github had it collapsed. Just inline the common styles